### PR TITLE
Generate pins from SVG (#5)

### PR DESCRIPTION
**Pull request by mikrut**
Pin information (such as net and alignment) is taken from SVG and added to a symbol file.
In symbol configuration file (yaml) default net numbers can be reassigned using ``pinout`` section.

I don't yet understand how to use ``valign`` and ``halign`` with multiple horizontal/vertical pins in SVG file. So currently they are used only to specify pin direction.
As I understand, these parameters are more useful for elements generated from YAML config (e.g. various integrated circuits), not from SVG image (in which each pin is drawn separately and already has its coordinates specified).

Should we treat single pin drawn in SVG as multiple pins if there are several numbers for its net given in YAML config? In that case we probably may use original qeda settings for pin spacing, something like:
```yaml
space:
  pin: 2
```

### CODE REVIEW

File: src/generators/kicad.rs [hunk 1]
```diff
@@ -123,6 +124,57 @@ impl fmt::Display for Visibility {
     }
 }
 
+impl ToLetter for ElectricKind {
+    fn to_letter(&self) -> char {
```
Comment (shamilsan) [inline src/generators/kicad.rs]: Why to not implement the `fmt::Display` trait here?

File: src/pin.rs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    Right,
+    Left,
+}
+
+#[derive(Debug)]
+pub struct Pin {
+    pub pos: Point,
+    pub length: f64,
+    pub net: String,
+    pub number: u32,
```
Comment (shamilsan) [inline src/pin.rs]: `number` may be a `String` for BGA packages

**Comment by shamilsan** [comment]
When multiple pins are defined in YAML config for non-IC (SVG-based) symbols, the pin line from the SVG picture is to be replaced by the plain line. Also, the joining line for multiple pins is to be added. I agree we could use the pin space parameter from the config file but we need to align it to the grid. Let's leave multiple pins feature to the future.

File: src/generators/kicad.rs (hunk 1 above)
Comment (mikrut) [reply src/generators/kicad.rs]: Good point! Looks like ``ToLetter`` trait isn't that useful, so I removed it completely.

File: src/pin.rs (hunk 2 above)
Comment (mikrut) [reply src/pin.rs]: Whoa. Thank you. That was a little bit unexpected. Made it a ``String`` too.

**Pull request merged**